test/Farmer/addLocation.test.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
+import nock from "nock";
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/JoeMindo/USSDTest/pullRequest?prid=8695127)
```suggestion
import nock from 'nock';
```